Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM golang:1.13-alpine3.10
+FROM golang:1.13-alpine3.12
```

Comment:
Would it be possible to bump golang too?

Reply:
Bumping golang could cause some breaking changes. This PR is just to address the CVE so we are not doing that. Anyway, the latest release also has this older version only: https://github.com/mintel/dex-k8s-authenticator/blob/v1.4.0/Dockerfile

So, we could wait for the next release and sync it with the upstream since the upstream master has the version updated.